src/components/CustomListItem.vue

```diff
@@ -14,7 +16,7 @@ const emits = defineEmits<{
 
     <textarea
       class="font-mono p-2 w-full resize-y m-0 outline-none -mb-1 min-h-[5rem]"
-      :placeholder="'Enter glob patterns:\n*.lock\n**/vendor/**'"
+      :placeholder="t('options_customLists_customRepoPlaceholder')"
```

Comment:
Using underscores make it hard to select individual parts of the key. Double clicking the string selects everything.

But I want to avoid anything at runtime like `.replaceAll('.', '_')` that could slow things down.